build_image.sh

```diff
@@ -59,29 +59,56 @@ read_value build_image_offer ".build.image_offer"
 read_value build_image_sku ".build.image_sku"
 read_value build_vm_size ".build.vm_size"
 read_value cyclecloud_version ".cyclecloud_version"
+read_value user_assigned_identity_client_id ".user_assigned_identity_client_id"
+read_value cyclecloud_package_name ".cyclecloud_package_name"
+read_value repo_stream ".repo_stream"
 
 timestamp=$(date +%Y%m%d-%H%M%S)
 
 # run packer
 pushd ./packer
-packer_log=../packer-output-$timestamp.log
-packer init ./ccmarketplace.pkr.hcl | tee $packer_log
-packer build \
-    -var location=$location \
-    -var subscription_id=$subscription_id \
-    -var resource_group=$resource_group \
-    -var build_resource_group_name=$build_resource_group_name \
-    -var virtual_network_name=$virtual_network_name \
-    -var virtual_network_subnet_name=$virtual_network_subnet_name \
-    -var virtual_network_resource_group_name=$virtual_network_resource_group_name \
-    -var image_name=$build_image_name \
-    -var image_publisher=$build_image_publisher \
-    -var image_offer=$build_image_offer \
-    -var image_sku=$build_image_sku \
-    -var vm_size=$build_vm_size \
-    -var cyclecloud_version=$cyclecloud_version \
-    . \
-    | tee -a $packer_log
+packer_log=packer-output-$timestamp.log
+packer init ./ccmarketplace.pkr.hcl | tee ../logs/$packer_log
+if [ "$repo_stream" == "insiders-fast" ]; then
```

Comment:
Let's eliminate the if and just pass the repo_stream as a -var (whether it's "" or "insiders")